### DISCUSSION TITLE
Access violation when converting 10000+ images

### DISCUSSION THREAD

**Discussion by alanccw** [ImageMagick/ImageMagick#7323] [Help]
### ImageMagick version

7.0.8-68

### Operating system, version and so on

Windows XP, Windows 7

### Description

Hi,

I try to use MagickReadImage and MagickWriteImage to convert more than 10000+ images from different formats, such as JPG, PSD, PNG, etc. to .BMP. And during the process, from time to time, I will get "Access Violation" error.

How to prevent this and start over?

**Comment by fmw42** [comment]
What was the exact error message?  Have you made sure that your policy.xml file is set up adequately?  Have you cleared out your /tmp or /temp directory?

**Comment by dlemstra** [comment]
You probably want to upgrade to the most recent version. We are no longer building our project with Windows XP compatibility so that might be a problem for you when upgrading. Windows 7 should still be supported but you will need to take a version earlier or wait for the next version that restores Windows 7 support.
